scripts/checks.sh

```diff
@@ -407,10 +407,10 @@ function check_balenaOS()
 			-H "Content-Type: application/json" \
 			-H "Authorization: Bearer ${DEVICE_API_KEY}" \
 			"${API_ENDPOINT}/v5/release?\$expand=release_tag,belongs_to__application,contains__image/image&\$filter=(belongs_to__application/any(a:a/device_type%20eq%20'${SLUG}'))%20and%20(release_tag/any(rt:(rt/tag_key%20eq%20'version')%20and%20(rt/value%20eq%20'${VERSION}')))" \
-			| jq -r "[.d[] | select(.release_tag[].value == (\"${VARIANT:-production}\" | ascii_downcase))] | length")
+			| jq -r "[.d[] | select(.release_tag[].value == (\"${VERSION}\" | ascii_downcase))] | length")
```

Comment:
This filter serves no purpose with the change you have, as we have already filtered by version in the pine query.

Reply:
@thgreasi I think we want something similar to your commit here?
https://github.com/balena-os/balenahup/commit/139296f35e02ce04e51063f5de1d25388fd95788

Reply:
Indeed @klutchell that jq filter is redundant.